pontoon/teams/views.py

```diff
@@ -77,6 +78,25 @@ def ajax_info(request, locale):
     })
 
 
+@require_POST
+@permission_required_or_403('base.can_manage_locale', (Locale, 'code', 'locale'))
+@transaction.atomic
+def ajax_update_info(request, locale):
+    description = request.POST.get('team_info', None)
+    tags = bleach.ALLOWED_TAGS[:]
+    # allow <p>
+    tags.extend(('br', 'p'))
+    attrs = bleach.ALLOWED_ATTRIBUTES.copy()
+    # allow <a target="">
+    attrs['a'].append('target')
+    description = bleach.clean(
+        description, tags=tags, attributes=attrs, strip=True)
+    _locale = get_object_or_404(Locale, code=locale)
```

Comment:
I'm not sure what's the best practice for naming variables like this. I'm not a big fan of the `_` prefix. I'd probably use `l`, but that's ugly either.